src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -138,14 +138,14 @@ def get_asic_id_from_name(asic_name):
         raise ValueError('Unknown asic namespace name {}'.format(asic_name))
 
 
-def get_current_namespace():
+def get_current_namespace(pid=None):
     """
     This API returns the network namespace in which it is
     invoked. In case of global namepace the API returns None
     """
 
     net_namespace = None
-    command = ["/bin/ip netns identify", str(os.getpid())]
+    command = ["sudo /bin/ip netns identify {}".format(str(os.getpid()) if not pid else str(pid))]
```

Comment:
I don't think you need the `str()` conversions here.

Reply:
@jleveque thanks updated.  Works fine and do not need str() conversion